### ISSUE TITLE
help with mining

### ISSUE THREAD

**Issue by FrazeDev** [UnamSanctam/SilentCryptoMiner#1909]
hello, could you help me start mining ravecoin on gpus i can't figure it out how to setup it

**Comment by UnamSanctam** [comment]
You select KawPow as the algorithm and then enter a Ravencoin pool and wallet address.

**Comment by FrazeDev** [comment]
> You select KawPow as the algorithm and then enter a Ravencoin pool and wallet address.

can you suggest any pools ?

**Comment by UnamSanctam** [comment]
2miners maybe.

**Comment by FrazeDev** [comment]
> 2miners maybe.

and do you know a wallet that has perm link because idk how to use zencore

**Comment by UnamSanctam** [comment]
Not sure what you mean by perm link, but all multi-wallets support Ravencoin.

**Comment by FrazeDev** [comment]
> Not sure what you mean by perm link, but all multi-wallets support Ravencoin.

is guarda wallet good ?

**Comment by UnamSanctam** [comment]
Don't know, I haven't used it (though I haven't used most wallets).

**Comment by FrazeDev** [comment]
> Don't know, I haven't used it (though I haven't used most wallets).

so what wallet do you personally use?



and these settings would work on rave 
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/72148172/8a83d5fe-85c1-4d12-821a-89e1599abca8)
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/72148172/f78a7f01-bc9e-4736-851e-f7b6ac87d319)

**Comment by UnamSanctam** [comment]
Usually I use ZelCore as a multi-wallet, otherwise things like MetaMask or Exodus work as well. Those settings look good but you'll need to use the RVN pool `rvn.2miners.com:16060` instead of the ETC pool.

**Comment by FrazeDev** [comment]
> Usually I use ZelCore as a multi-wallet, otherwise things like MetaMask or Exodus work as well. Those settings look good but you'll need to use the RVN pool `rvn.2miners.com:16060` instead of the ETC pool.


https://github.com/UnamSanctam/SilentCryptoMiner/assets/72148172/a1303096-beb0-4c16-9a54-bfc58fb833a2

do i put this one in the miner ? 
and will the wallet change

**Comment by UnamSanctam** [comment]
Yes that is correct, and no the wallet address will never change. The ZelCore wallet has multiple wallets so you can use the "Mining" wallet instead just to keep track better (it's just a name, you can set it to anything).

**Comment by FrazeDev** [comment]
> Yes that is correct, and no the wallet address will never change. The ZelCore wallet has multiple wallets so you can use the "Mining" wallet instead just to keep track better (it's just a name, you can set it to anything).

what you mean mining wallet

**Comment by UnamSanctam** [comment]
At the top in your video there are different wallets, such as "Payments", "Income", "Mining" and more, so using the "Mining" wallet might make more sense. Though the wallets names are just nicknames and can be whatever.

**Comment by FrazeDev** [comment]
any idea why this miner didin't connect to the pool but other miners dis 
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/72148172/da3ba327-ab3a-44e4-8463-c34cc3071503)

**Comment by UnamSanctam** [comment]
Either their network (something local, their ISP or government) is blocking the pool server, or the pool server has blocked them.

**Comment by FrazeDev** [comment]
> Either their network (something local, their ISP or government) is blocking the pool server, or the pool server has blocked them.

so theres no way to fix it ?

**Comment by UnamSanctam** [comment]
Not really since the issues is on their side, you can set just that miner to another pool though. If there were more different pool servers then you could just set it to another pool server (for example 2miners RVN has an US and Asia pool server as well) but 2miners XMR only has one pool server.

**Comment by FrazeDev** [comment]
the issue is that it connected to rvn.2miner it doesn't connect to the xmr

**Comment by UnamSanctam** [comment]
Yes the specific XMR pool server is probably blocked, I meant that the issue is that 2miners only has one XMR pool server so you can't switch that like how you can for RVN. So to mine with that computer you would either need to mine to another pool or mine via a proxy using something like xmrig-proxy (but you have to run that yourself).
